constants.py

```diff
@@ -0,0 +1,17 @@
+
+# Game configuration constants
+INITIAL_LIFE_LEVEL = 100
+MAX_ROW = 4
+MAX_COL = 4
+HONEY_AMOUNT = 6
```

Comment:
You can calculate this using the amount of bears

Reply:
yes, I wanted to experiment with different sizes and amount of bears not depending on the field size